server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java

```diff
@@ -97,6 +100,17 @@ protected void doInit(JobContext context) {
                 PublishSqlPlanJobReq.class);
         this.connectionSession = generateSession();
         this.executor = connectionSession.getSyncJdbcExecutor(ConnectionSessionConstants.CONSOLE_DS_KEY);
+        long timeoutUs = TimeUnit.MILLISECONDS.toMicros(parameters.getTimeoutMillis());
+        if (timeoutUs < 0) {
+            throw new IllegalArgumentException(
```

Comment:
the error message is not correct, may simplify as 'Invalid timeout settings', and please use PreConditions  instead

Reply:
Have been modified.